github/github_actions/add_build_matrix.rb

```diff
@@ -23,6 +23,10 @@ def workflow_files
 
 def run_on_tw(content, job)
   content.gsub!(
+    # (?:(?!distro).)* = anything except "distro", this is a non-greedy version of .*
+    # (: starts a non-capturing group
+    # (?! is a negative lookahead assertion
+    # https://ruby-doc.org/core-2.5.0/Regexp.html#class-Regexp-label-Anchors
```

Comment:
```suggestion
    # https://ruby-doc.org/core-2.5.0/Regexp.html#class-Regexp-label-Anchors
    # (Next time try a simpler lazy match, #{job}:.*?distro:
```

If you really want just a lazy `.*` then `.*?` should work:

> Repetition is greedy by default: as many occurrences as possible are matched while still allowing the overall match to succeed. By contrast, lazy matching makes the minimal amount of matches necessary for overall success. A greedy metacharacter can be made lazy by following it with ?.

I don't want to drag out a helper script that has already served its purpose, so I suggest just adding a comment to remind us the next time.